### DISCUSSION TITLE
Problema na instalação do MAPOS

### DISCUSSION THREAD

**Discussion by DataPowerTI** [RamonSilva20/mapos#2453] [General]
Boa noite, acabei de fazer a instalação do mapos, porém quando vou tentar acessar o sistema: exemplo.datapowerti.com.br ele redireciona duplicando: https://exemplo.datapowerti.com.br/exemplo.datapowerti.com.br/index.php/login
Alguém consegue me ajudar?

**Comment by Pr3d4dor** [comment]
Seu .env deve estar configurado errado o APP_URL.

**Reply by DataPowerTI** [reply]
onde eu corrijo isso?
onde eu encontro o lugar certo pra corrigir? consegue me ajudar?

**Reply by Pr3d4dor** [reply]
No próprio arquivo .env vc pode ver isso

**Comment by DataPowerTI** [comment]
Conseguui encontrar o local onde estava o erro, adicionei o https:// antes do endereço, assim não duplicou.

**Reply by juniorlima14** [reply]
onde é esse erro?
